src/Samples/DesktopClient/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I presume this is from VSCode? Can you remove this from your PR. While I do see benefit in having this, it should be done purposefully and across the entire repo for every project where appropriate rather than a single adhoc project.

Reply:
I use VS2019, I'll post edit accordingly.